qcodes/station.py

```diff
@@ -355,11 +369,19 @@ def update_load_instrument_methods():
                         self.load_instrument, identifier=instrument_name))
                     self._added_methods.append(method_name)
                 else:
-                    log.warning(f'Invalid identifier: ' +
-                                f'for the instrument {instrument_name} no ' +
-                                f'lazy loading method {method_name} could ' +
+                    log.warning(f'Invalid identifier: '
+                                f'for the instrument {instrument_name} no '
+                                f'lazy loading method {method_name} could '
                                 'be created in the Station.')
-        self._config = YAML().load(config)
+        yaml = YAML().load(config)
+        with open(get_schema_file_path()) as f:
+            schema = json.load(f)
+        try:
+            jsonschema.validate(yaml, schema)
+        except jsonschema.exceptions.ValidationError as e:
+            warnings.warn(e.message, ValidationWarning)
```

Comment:
Do we log this?

Reply:
👍 All warnings get logged.
https://github.com/QCoDeS/Qcodes/blob/72890d0348cf4a141e8e879b950fdd4f4bdb6440/qcodes/logger/logger.py#L214